manifests/fedora-coreos-base.yaml

```
@@ -157,6 +157,8 @@ packages:
   - kbd
   # Parsing/Interacting with JSON data
   - jq
+  # nvme-cli for udev rules
```

Since we're adding it independent of the udev rules let's just make the comment more generic. Something like:

```suggestion
  # nvme-cli for managing nvme disks
```

---

Sure, Thank you for the review @dustymabe .